src/main/java/module/lineup/assistant/LineupAssistant.java

```diff
@@ -53,159 +53,250 @@ public final boolean isPlayerInStartingEleven(int spielerId, Vector<MatchLineupP
 	/**
 	 * Assistant to create automatic lineup
 	 *
-	 * @param lPositions:              list of positions to be filled
+	 * @param allPositions:              list of positions to be filled
 	 * @param lPlayers:                list of available players
 	 * @param sectorsStrengthPriority: priority in sector strength (e.g. MID-FOR-DE)
 	 * @param bForm:                   whether or not to consider the form
 	 * @param idealPosFirst:           whether or not to consider best position first
-	 * @param bInjured:                whether or not to consider injured player
-	 * @param bSuspended:              whether or not to advanced suspended player
+	 * @param ignoreInjured:                whether or not to consider injured player
+	 * @param ignoreSuspended:              whether or not to advanced suspended player
 	 * @param weather:                 Actual weather
 	 */
-	public final void doLineup(List<MatchLineupPosition> lPositions, List<Player> lPlayers,
-							   byte sectorsStrengthPriority, boolean bForm, boolean idealPosFirst, boolean bInjured,
-							   boolean bSuspended, Weather weather) {
+	public final void doLineup(List<MatchLineupPosition> allPositions, List<Player> lPlayers,
```

Comment:
Here I am thinking of working myself on a unit test framework that would allow us to test these things.

Reply:
Arguments form and weather are currently not used.